tests/integration/example/integration.sh

```diff
@@ -26,35 +26,47 @@ function process_result() {
     fi
 }
 
-# Build mixer binary
-bazel build //mixer/cmd/mixs:mixs
-MIXER_BINARY=$(pwd)/bazel-bin/mixer/cmd/mixs/mixs
+# If GOPATH is not set by the env, set it to a sane value
+GOPATH ?= $(shell cd ../../..; pwd)
+export GOPATH
 
-# Get fortio
-go get -u istio.io/fortio
+# Build mixer binary
+make mixs
+MIXER_BINARY=$(make where-is-out)/mixs
 
```

Comment:
If you add
<pre>ENVOY_BINARY=$(make where-is-out)/envoy
</pre>
here then you can probably avoid the need to source istio.VERSION and download envoy.  Well, I am assuming that "make mixs" implicitly does a "make init" or "make depend" (they're currently synonyms), which it might not do anymore (but if it doesn't then I can't see how mixs compiles since it probably relies on the vendors that are pulled in by make init/depend).  Worse case is changing "make mixs" to "make init mixs", but I'm guessing that if mixs is already compiling successfully then the init is already getting triggered somewhere.

Reply:
You are right, that works!